docs/api/browser-window.md

```diff
@@ -1810,14 +1810,16 @@ there is only one tab in the current window.
 
 Adds a window as a tab on this window, after the tab for the window instance.
 
-#### `win.setVibrancy(type)` _macOS_
+#### `win.setVibrancy(type[, animate])` _macOS_
 
 * `type` string | null - Can be `appearance-based`, `light`, `dark`, `titlebar`,
   `selection`, `menu`, `popover`, `sidebar`, `medium-light`, `ultra-dark`, `header`, `sheet`, `window`, `hud`, `fullscreen-ui`, `tooltip`, `content`, `under-window`, or `under-page`. See
   the [macOS documentation][vibrancy-docs] for more details.
+* `animate` boolean (optional)
```

Comment:
An options object might be better since there are lots of other vibrancy related things that could be added in future.

Reply:
I agree with @zcbenz.  This should be an options object for future proofing.

Reply:
refactored API to use options object